tests/RobotFramework/tests/cumulocity/configuration/plugins/hello

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage() {
+    echo "Usage: $0 <command> [args...]"
+    echo "Commands:"
+    echo "  list                                    List all config types supported by this plugin"
+    echo "  get <type> <target-file-path>           Get config for the specified type and write to the target path"
+    echo "  set <type> <new-config-path>            Set config for the specified type with the new config path"
```

Comment:
The contract is not clear as this comment is not aligned with the code.

I understand the comment as:

```suggestion
    echo "  get <type> <target-file-path>          Copy the config for the specified type to the target path"
    echo "  set <type> <new-config-path>        Update the config for the specified type from new config path"
```

But the script behaves as:


```suggestion
    echo "  get <type> <target-file-path>           Copy the config for the specified type to stdout"
    echo "  set <type> <new-config-path>         Update the config for the specified type from new config path"
```

Reply:
Resolved by a16fff1cbf